hazelcast/src/main/java/com/hazelcast/monitor/impl/HotRestartStateImpl.java

```diff
@@ -28,25 +28,38 @@
 
     private BackupTaskStatus backupTaskStatus;
 
+    private boolean isHotBackupEnabled;
+
     public HotRestartStateImpl() {
     }
 
     public HotRestartStateImpl(BackupTaskStatus backupTaskStatus) {
```

Comment:
You can remove this constructor since I guess it is only used in tests now. Change the tests to use the new one.